src/nix/hive/eval.nix

```diff
@@ -512,7 +512,7 @@ let
   deploymentConfig = lib.mapAttrs (name: eval: eval.config.deployment) nodes;
 
   deploymentConfigSelected = names:
-    listToAttrs (map (name: { inherit name; value = nodes.${name}.config.deployment; }) names);
+    listToAttrs (map (name: { inherit name; value = builtins.removeAttrs nodes.${name}.config.deployment ["nixpkgs"]; }) names);
```

Comment:
This is necessary (and probably also above in `deploymentConfig`) because evaluating the deployment config would try to evaluate Nixpkgs, which has its very nice "_AAAAAwhatever" package that throws an error saying you probably shouldn't evaluate all of Nixpkgs.